Bosonic Fitting (#72)
qutip/qutip-tutorials: tutorials-v5/heom/heom-1d-spin-bath-model-ohmic-fitting.md

## Helper functions

> Let's define some helper functions for plotting the resutls

Reviewer: Suggestion: delete

Reviewer: Github is formatting this weirdly, I meant to delete lines 73-76 since there are no helper functions anymore

where $a, b$ and $c$ are the fit parameters and each is a vector of length $k$.

+++

> With the spectral density approximation $J_{\mathrm approx}(w; a, b, c)$ implemented above, we can now perform the fit and examine the results. This can be done quickly using the FitSpectral bath, which takes the target spectral density as an array and fits it to the series of **k** underdamped harmonic oscillators with the Meier-Tannor form

Reviewer: ```suggestion
This type of fit can be performed quickly using the FitSpectral bath, which takes the target spectral density as an array and fits it with a series of **k** underdamped harmonic oscillators with the Meier-Tannor form.
```
